docs/content/development/datasketches-aggregators.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: doc_page
+---
+
+## DataSketches aggregator
+Druid aggregators based on [datasketches]()http://datasketches.github.io/) library. You would ingest one or more metric columns using either `sketchBuild` or `sketchMerge` aggregators. Then, at query time, you can use `sketchMerge` with appropriate post aggregators described below. Note that sketch algorithms are approxiate, see details in the [datasketches doc](http://datasketches.github.io/docs/theChallenge.html).
+
+#### aggregators to use at ingestion time
+
+##### non-sketch input data
+You can ingest data to druid and build the sketch objects which are later used at the time of querying to compute uniques.
```

Comment:
My understanding is a sketch refers to a method of calculation whereby you generate noise from data in a consistent way, then use properties of the noise to describe the data in some approximate fashion. HyperLogLog is the sketch algorithm used bye the hyperUniques and cardinality aggregators to estimate the unique counts of a set of hashable items.